pom.xml

```diff
@@ -39,6 +39,8 @@
     <module>hawkular-wildfly-monitor</module>
     <module>hawkular-wildfly-monitor-feature-pack</module>
     <module>hawkular-agent-itest-parent</module>
+    <module>extension-installer</module>
+    <module>hawkular-agent-installer</module>
```

Comment:
rename this to match the name we use elsewhere "hawkular-wildfly-monitor" - so call this module something like "hawkular-wildfly-monitor-installer"